wireshark/Dockerfile

```diff
@@ -1,75 +1,144 @@
-# Define the wireshark version to be baked in.
-ARG WIRESHARK_VERSION=3.4.9
+# This Dockerfile builds a Wireshark image with Open Quantum Safe (OQS) support.
+# By integrating OQS, the resulting Wireshark build is capable of
+# analyzing and handling post-quantum cryptographic protocols.
```

Comment:
worthwhile adding a version LABEL to distinguish (future) releases?

Reply:
Just check out the other demos: They carry a version label that's updated manually each time a substantial functionality change occurs. We forgot to add it in the first iteration, so value "2" is probably in order now.

Reply:
Why did you mark this comment resolved, @Hayyaaf ? Normal etiquette is that the person opening an issue closes it. In this case, I still don't find a version label in the Dockerfile. Can you please point me to it in case I overlooked it?